src/components/action_buttons/PatientsButtonsRenderer.jsx

```diff
@@ -0,0 +1,50 @@
+import { Button, Box } from "@mui/material";
+import DeleteIcon from "@mui/icons-material/Delete";
+import NoteAltIcon from "@mui/icons-material/NoteAlt";
+import InventoryIcon from "@mui/icons-material/Inventory";
+import VisibilityIcon from "@mui/icons-material/Visibility";
+function PatientsButtonsRenderer() {
+  const buttonStyle = () => ({
```

Comment:
When you are setting up styles like this, you can just use objects instead of functions. Then you just set the style to the object instead of calling a function